code/__DEFINES/mob.dm

```diff
@@ -54,6 +54,8 @@
 #define DEFAULT_MARKING_STYLES list("head" = "None", "body" = "None", "tail" = "None") //Marking styles. Use instead of initial() for m_styles.
 #define DEFAULT_MARKING_COLOURS list("head" = "#000000", "body" = "#000000", "tail" = "#000000") //Marking colours. Use instead of initial() for m_colours.
 
+#define OXYCONCEN_PLASMEN_IGNITION 0.01 //1% oxygen in the air is all it takes to light up a poorly clothed Plasmaman.
```

Comment:
@Fox-McCloud Where are you getting that number from? I checked in the atmospherics defines and it says ```PLASMA_MINIMUM_OXYGEN_NEEDED ``` ```        ``` ```2```